.github/workflows/integrationTest.yml

```diff
@@ -126,7 +126,6 @@ jobs:
       ec2_gpu_matrix: ${{ steps.set-matrix.outputs.ec2_gpu_matrix }}
       ec2_linux_matrix: ${{ steps.set-matrix.outputs.ec2_linux_matrix }}
       ec2_performance_matrix: ${{steps.set-matrix.outputs.ec2_performance_matrix}}
-      ec2_windows_matrix: ${{ steps.set-matrix.outputs.ec2_windows_matrix }}
```

Comment:
Since this is in the yaml, can't we just comment this out like other parts of the file?